wildfly-base/src/main/docker/cli-scripts/ers_datasources.cli

```diff
@@ -0,0 +1,48 @@
+set usm_ers_pw=${db_password_usm_ers}
+set ers_pw=${db_password_ers}
+
+embed-server --server-config=standalone-full.xml
+
+# USM2 configured for ERS
+xa-data-source add \
+	--name=jdbc/USM2_ers \
```

Comment:
Isn't the 2 in USM2_ers a bit unnecessary when we have the suffix _ers?

Reply:
The 2 actually comes from the original datasource which is called jdbc/USM2 - I guess the 2 is second generation or something